packages/playground/src/app.js

```diff
@@ -581,14 +581,6 @@ class Playground extends Component {
                     id="theme"
                     href={this.state.stylesheet || ""}
                   />
-                  {theme === "antd" && (
```

Comment:
apparently we don't actually need this, antd theme seems to work fine without it

Reply:
I wonder why that is? Maybe a recent upgrade to `antd` made the need for it irrelevant?  As long as the theme still works in the playground... ship it

Reply:
yeah not sure haha... I'm happy to ship it because it's just the playground rather than an actual published npm package that people depend on.